testcases/kernel/controllers/README

```diff
@@ -79,3 +79,11 @@ CONFIG_FAIR_GROUP_SCHED=y
 CONFIG_RT_GROUP_SCHED=y
 # CONFIG_USER_SCHED is not set
 CONFIG_CGROUP_SCHED=y
+CONFIG_CGROUP_CPUACCT=y
+CONFIG_CGROUP_NET_PRIO=y
+CONFIG_CGROUP_PIDS=y
+
+Note
+----
+CONFIG_CGROUP_NS is only for kernel 2.6.24 - 2.6.39
+CONFIG_USER_SCHED is only for kernel 2.6.25 - 2.6.33
```

Comment:
If nothing else this should be in a separate patch. Ideally the changes should be divided into patches so that each patch contains one logical well defined change.